src/main/java/Awex.java

```diff
@@ -0,0 +1,130 @@
+import java.util.*;
```

Comment:
Perhaps you could try to avoid the use of wildcard imports here as it could lead to import conflicts.